custom_components/smartenergy/coordinator.py

```
@@ -47,6 +48,45 @@
+
+    cheaper: int
+    """Echt günstigere Intervalle desselben Tages."""
+
+    equal: int
+    """Gleich teure Intervalle, das eigene eingeschlossen (also mindestens 1)."""
+
+    quantile: float
+    """Mittelrang-Quantil in Prozent, 0 % = günstigstes Intervall.
+
+    ``(cheaper + equal / 2) / count``. Der Mittelrang teilt einen Gleichstand
+    hälftig und bleibt dadurch symmetrisch – der Tagesdurchschnitt liegt bei
```

Ruff (0.16.2)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)